packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -317,13 +317,30 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
       }
       _extent._currentExtent.value = _extent.initialExtent;
     }
+    _child = widget.builder(context, _scrollController);
+  }
+
+  @override
+  void didUpdateWidget(DraggableScrollableSheet oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    _child = widget.builder(context, _scrollController);
```

Comment:
Maybe add a quick comment here reminding us in the future why we have to call this unconditionally (and not just when builder changes)?

Reply:
Done.